src/components/App/Header/UserMenu/index.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import styles from './UserMenu.module.scss';
+
+const UserMenu = props => {
+  return (
+    <nav role="navigation" className={styles.userMenuWrapper}>
+      <ul className={styles.menuList}>
+        <li>
+          <span className={styles.myAccount}>
+            <span>My account</span>
+          </span>
+          <ul className={styles.dropdown}>
+            <li><Link to="/watchlist">Watchlist</Link></li>
+            <li><a href="#">Sign out</a></li>
```

Comment:
As user I cannot sign in so I don't need sign out button, right? I know it is probably 'for the future feature'.

Reply:
Right, but maybe it's indeed better to take it away for now. :)